source/NVDAObjects/window/_msOfficeChart.py

```diff
@@ -0,0 +1,972 @@
+				elif direction=="next": self.activeElement = self.activeElement.next
+			self.activeElement.select() 
+			eventHandler.queueEvent("gainFocus", self.activeElement)
+							
+	def script_previousElement(self,gesture):
+		self.navigateToElement("previous")
+	script_previousElement.canPropagate=True
+	
+	def script_nextElement(self,gesture):
+		self.navigateToElement("next")
+	script_nextElement.canPropagate=True
+
+	__gestures = {
+				"kb(laptop):leftArrow":"previousElement",
+				"kb(desktop):leftArrow":"previousElement",
+				"kb(laptop):rightArrow":"nextElement",
+				"kb(desktop):rightArrow":"nextElement",
+	}
+
+class OfficeChart(OfficeChartElementList):
+
+	role=controlTypes.ROLE_CHART
+
+	def __init__(self,windowHandle, officeApplicationObject, officeShapeObject,keyIndex=0):
+		super(OfficeChart,self).__init__(windowHandle=windowHandle  , officeChartObject = officeShapeObject.Chart )
+		self.officeApplicationObject=officeApplicationObject
+		try:
+			seriesCount=self.officeChartObject.SeriesCollection().Count
+		except:
+			seriesCount=None
+		if seriesCount:
+			for i in xrange(seriesCount):
+				self.addElement( OfficeChartElementSeries(windowHandle=self.windowHandle, officeChartObject = self.officeChartObject , elementID = xlSeries , arg1 = i +1 ) , self) 
+
+		self.addElement( OfficeChartElement(windowHandle=self.windowHandle, officeChartObject = self.officeChartObject ) , self )
+		try:
+			self.officeChartObject.Select()
+		except:
+			pass		
+		
+	def _get_name(self):
+		if self.officeChartObject.HasTitle:
+			name=self.officeChartObject.ChartTitle.Text
+		else:
+			name=self.officeChartObject.Name
+		#find the type of the chart
+		chartType = self.officeChartObject.ChartType
+		chartTypeText = chartTypeDict.get(chartType,
+                # Translators: Reported when the type of a chart is not known.
+                                _("unknown"))
+		# Translators: Message reporting the title and type of a chart.
+		text=_("Chart title: {chartTitle}, type: {chartType}").format(chartTitle=name, chartType=chartTypeText)
+		return text
+
+	def _get_description(self):
+		count = self.officeChartObject.SeriesCollection().count
+		text=""
+		if count>0:
+			if count == 1:
+				# Translators: Indicates that there is 1 series in a chart.
+				seriesValueString = _( "There is 1 series in this chart" )
+			else:
+				# Translators: Indicates the number of series in a chart where there are multiple series.
+				seriesValueString = _( "There are total %d series in this chart" ) %(count)
+				for i in xrange(1, count+1):
+					# Translators: Specifies the number and name of a series when listing series in a chart.
+					seriesValueString += ", " + _("series {number} {name}").format(number=i, name=self.officeChartObject.SeriesCollection(i).Name)
+				text += seriesValueString
+		else:
+			# Translators: Indicates that there are no series in a chart.
+			text +=_("No Series defined.")
+		return text
+
+	def getChartSegment(self):
```

Comment:
Is this different to OfficeChartElementBase.getChartSegment? if not then this can be removed.